### PR TITLE
feat(cli): generate REAMDE file in amplify dir

### PR DESCRIPTION
**Pull request by watilde**
*Issue* https://github.com/aws-amplify/amplify-cli/issues/5600

*Description of changes:*
When `amplify init` is triggered, it generates `README.md` file under `amplify` directory that gives users a guide of how to use it. We can use this patch as a starting point to provide README and improve the content going forward based on feedbacks from community.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import fs from 'fs-extra';
+const ReadMeContent = `# Getting Started with amplify-cli
```
Comment (renebrandel) [inline packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts:2]: with Amplify CLI

File: packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+This directory was generated with [amplify-cli][] command. This README helps you to find resources you are looking for.
```
Comment (renebrandel) [inline packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts:3]: Amplify CLI [docs.amplify.aws/cli]

Comment (renebrandel) [reply packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts:3]: I'd shorten this section to `This directory was generated by [Amplify CLI][docs.amplify.aws/cli] command.`

File: packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## About CLI
```
Comment (renebrandel) [inline packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts:5]: Shorten this to 
```
Helpful resources:
- Amplify documentation: docs.amplify.aws
- Amplify CLI documentation: docs.amplify.aws/cli
- More details on this folder & generated files: https://docs.amplify.aws/cli/reference/files
- Join Amplify's Discord community: dicord.gg/amplify
```

File: packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts (hunk 2 above)
Comment (watilde) [reply packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts]: Great input, I've updated :)

File: packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts (hunk 3 above)
Comment (watilde) [reply packages/amplify-cli/src/extensions/amplify-helpers/docs-manager.ts]: Thanks for the inputs! For the discord link, I've replaced with community link as it's a parent page of Discord invitation link and there is a possibility to add more pages or chat links for different regions on the community page.

E.g. Users in Japan may not join the Discord because of language barrier, but will join community Slack will be led by community members.

**Comment by watilde** [comment]
@kaustavghosh06 @attilah This is just a friendly reminder that we are waiting for your review. Hope we can ship this out together :)

**Review by palpatim — APPROVED** [review]

**Comment by renebrandel** [comment]
Reviewed this with @undefobj & @palpatim also and it looks good! Let's get this merged! cc @attilah

**Review by renebrandel — APPROVED** [review]

**Comment by renebrandel** [comment]
@attilah one thing to just make sure of is that this new file is in conformity with source control recommendations & doesn't conflict for existing projects.
